integration/util/scenario/chain.js

```diff
@@ -197,16 +227,21 @@ class Chain {
   }
 
   async cullNotices() {
```

Comment:
TODO: Can we remove this now? We might need to remove earlier integration tests that depended on this function, but that's probably okay?